chartmogul/retry_request.py

```diff
@@ -19,7 +19,7 @@ def _retry_adapter(retries, backoff_factor):
         read=retries,
         connect=retries,
         status=retries,
-        method_whitelist=METHOD_WHITELIST,
+        allowed_methods=METHOD_WHITELIST,
```

Comment:
<details>
<summary>To fix this issue</summary>

<img width="715" alt="Screenshot 2023-03-14 at 11 48 55 AM" src="https://user-images.githubusercontent.com/8046636/224890590-5c47503b-38fb-4321-a820-8ec8268da2c3.png">

</details>